src/main/java/domain/dateTime/AttendanceDateTime.java

```diff
@@ -0,0 +1,84 @@
+package domain.dateTime;
+
+import static controller.AttendanceCommandController.SYSTEM_DATE_TIME;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+public class AttendanceDateTime implements Comparable<AttendanceDateTime> {
+    private static final String ATTENDANCE_DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm";
```

Comment:
```suggestion
    private static final String ATTENDANCE_DATE_TIME_FORMAT = "yyyy-M-d H:m";
```

이런 식으로 입력 format을 수정한다면 사용자가 한자리 숫자를 입력하더라도, `DateTimeParseException` 에러가 일어나지 않도록 할 수 있어요 :)
물론, 2자리 수로 입력을 제한할 수는 있겠지만, 한자리 수 입력을 허용하는 방식이 좀 더 사용자의 입력 방식에 유연하게 대응할 수 있을 것 같아요😺